SPECS/calamares/calamares.spec

```diff
@@ -136,101 +135,96 @@ rm -rf src/modules/{users,finished,welcome,partition,license}
 %setup -q -T -a 4 -D
 %setup -q -T -a 5 -D
 for module in users finished welcome partition license; do
-    mv "$module" src/modules/"$module"
+    mv "$module-%{version}" src/modules/"$module"
 done
 
 
 # Apply custom license config
 mv %{SOURCE20} src/modules/license/license.conf
-mv %{SOURCE24} src/modules/users/users.conf
 
-%patch 0 -p1
-%patch 1 -p1
-%patch 2 -p1
-%patch 3 -p1
-%patch 4 -p1
-%patch 5 -p1
+%patch -P 0 -p1
+#%patch3 -p1
```

Comment:
commented out patches.  Are they needed?

Reply:
See above, currently leaving these in as it is currently unknown if they are required changes or not due to the limited iso testing available currently for 3.0.